protected/humhub/tests/codeception/_support/HumHubDbTestCase.php

```diff
@@ -147,24 +146,51 @@ public function assertHasNotification($class, ActiveRecord $source, $originator_
 
     public function assertHasActivity($class, ActiveRecord $source, $msg = null)
     {
-        $activity = Activity::findOne(['class' => $class, 'object_model' => $source->className(), 'object_id' => $source->getPrimaryKey()]);
+        $activity = Activity::findOne([
+            'class' => $class,
+            'object_model' => $source->className(),
+            'object_id' => $source->getPrimaryKey(),
+        ]);
         $this->assertNotNull($activity, $msg);
     }
 
+    /**
+     * @return \Codeception\Module\Yii2|\Codeception\Module
+     * @throws \Codeception\Exception\ModuleException
+     */
+    public function getYiiModule() {
+        return $this->getModule('Yii2');
+    }
+
+    /**
+     * @deprecated $msg unused
+     * @see assertSentEmail
+     * @since 1.3
+     */
     public function assertMailSent($count = 0, $msg = null)
     {
-        return $this->getModule('Yii2')->seeEmailIsSent($count);
+        return $this->getYiiModule()->seeEmailIsSent($count);
+    }
+
+    /**
+     * @param int $count
+     * @throws \Codeception\Exception\ModuleException
+     * @since 1.3
+     */
+    public function assertSentEmail($count = 0)
+    {
+        return $this->getYiiModule()->seeEmailIsSent($count);
     }
 
     public function assertEqualsLastEmailSubject($subject)
     {
-        $message = $this->getModule('Yii2')->grabLastSentEmail();
+        $message = $this->getYiiModule()->grabLastSentEmail();
         $this->assertEquals($subject, str_replace(["\n", "\r"], '', $message->getSubject()));
     }
 
     public function allowGuestAccess($allow = true)
     {
-        if($allow) {
```

Comment:
shorter but obfuscated
`Yii::$app->getModule('user')->settings->set('auth.allowGuestAccess', $allow === true ? 1 : 0);`

Reply:
When you have some time you could add a phpdoc for allowGuestAccess ;-)